prow/gerrit/adapter/adapter_test.go

```diff
@@ -559,7 +559,7 @@ func createTestRepoCache(t *testing.T, ca *fca) (*config.InRepoConfigCache, erro
 	cache, err := config.NewInRepoConfigCache(
 		10,
 		ca,
-		config.NewInRepoConfigGitCache(cf))
+		cf)
```

Comment:
Call above should be one line.

Reply:
Just saw this. Cole already approved and I don't want to ask him to re-LGTM for such a small change. Will queue in my cleanup followups branch, thanks.

Reply:
See commit "style: prefer single line for function arguments" at #30428.